README.md

```diff
@@ -2,59 +2,109 @@
+Found the basic requirements too easy? Want to go the extra mile to impress us?
+Whatever the reason is, here are some optional ways in which you can give us a
+better sense of your approach, of your analytical and communication skills and
+of what you care about:
 
 - Add tests to your code.
-- Add more complex import status report (like a "percentage completed" estimation, or an ETA, for example).
-- We use and love open source software, and all OSS code must be properly documented. Add relevant documentation, beyond the “quickstart”, as you would for a publicly available OSS project.
+- Add more complex import status report (like a "percentage completed"
+  estimation, or an ETA, for example).
+- We use and love open source software, and all OSS code must be properly
+  documented. Add relevant documentation, beyond the “quickstart”, as you would
+  for a publicly available OSS project.
 
 ## 4. Things to keep in mind:
 
-- Your application and architecture can be as simple or as complex as you want, but be pragmatic and mindful of the trade-off between feature-completeness and complexity/performance/time to implement.
+- Your application and architecture can be as simple or as complex as you want,
+  but be pragmatic and mindful of the trade-off between feature-completeness and
+  complexity/performance/time to implement.
 - Assume that users are ok with imports not being instantaneous.
-- If you are not sure about a particular requirement/detail, make an educated guess and document your assumption.
-- This code challenge should match your CV. If you have experience with any tools that are perfect for this task, this is your opportunity to demonstrate that, and impress us.
-- **Please submit your ideas to us in 1 week (max)**. This will give us enough time to review your challenge with the rest of the team before the next interview. 
-- Based on the experiences of previous candidates, we believe **it will take you between 6 and 10 hours to complete the challenge**. If you see yourself allocating a lot more than that, you are including too much and/or overcomplicating it :)
+- If you are not sure about a particular requirement/detail, make an educated
+  guess and document your assumption.
+- This code challenge should match your CV. If you have experience with any
+  tools that are perfect for this task, this is your opportunity to demonstrate
+  that, and impress us.
+- **Please submit your ideas to us in 1 week (max)**. This will give us enough
+  time to review your challenge with the rest of the team before the next
+  interview.
+- Based on the experiences of previous candidates, we believe **it will take you
+  between 6 and 10 hours to complete the challenge**. If you see yourself
```

Comment:
Is this time still realistic or should remove any expected time since we already put a 1 week timeline

Reply:
The original purpose of this was to guide candidates in terms of expected scope - we don't want to risk appearing to inflict a 168hrs hackathon on them (and also, we wouldn't want them to deliver a very extensive project that then takes _us_ a 168hrs group hackathon to read and evaluate!)

Reply:
so tl;dr I'd keep this as guidance in terms of expected scope - if they're taking much longer, it could be ok if they want to, but it could be a red flag in terms of going overboard with the exercise